.github/workflows/linters.yml

```diff
@@ -0,0 +1,50 @@
+name: Linters
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+env:
+  python-version: "3.10"
+jobs:
+  black:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.4.0
+
+      - name: Set up Python ${{ env.python-version }}
+        uses: actions/setup-python@v2.3.1
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - run: pip install black
+
+      - run: black --check .
+
+  isort:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.4.0
+
+      - name: Set up Python ${{ env.python-version }}
+        uses: actions/setup-python@v2.3.1
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - run: pip install isort
+
+      - run: isort --check .
+
+  flake8:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.4.0
+
+      - name: Set up Python ${{ env.python-version }}
+        uses: actions/setup-python@v2.3.1
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - run: pip install flake8
+
+      - run: flake8 .
```

Comment:
Why would you run the linters on GH actions? This is exactly what pre-commit.ci is doing for us already, we don't need this at all in gh